kernel/packages/voice-chat-codec/VoiceCommunicator.ts

```diff
@@ -283,28 +283,27 @@ export class VoiceCommunicator {
     this.input?.workletNode.port.postMessage({ topic: topic })
   }
 
-  private createRTCLoopbackConnection(
-    currentRetryNumber: number = 0
-  ): { src: RTCPeerConnection; dst: RTCPeerConnection } {
+  private createRTCLoopbackConnection(retryNumber: number = 0): { src: RTCPeerConnection; dst: RTCPeerConnection } {
     const src = new RTCPeerConnection()
     const dst = new RTCPeerConnection()
 
-    let retryNumber = currentRetryNumber
+    // Apparently the RTCPeerConnection can be 'undefined' until the user accepts/blocks the microphone usage in the browser (https://github.com/webRTC-io/webrtc.io-client/issues/30#issuecomment-15991572)
+    // @ts-ignore
+    if (!dst || !src) {
```

Comment:
This condition will **_always_** return false.

There is NO WAY in that calling `new X` returns other than an object. It was a misinterpretation of the comment in the link.

That `if` should be removed